apps/web/components/ServiceInformation/TagList.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { FC } from 'react'
+import { useNamespace } from '@island.is/web/hooks'
+import {
+  Box,
+  Inline,
+  Tag,
+  Text,
+  GridColumn,
+  GridContainer,
+  GridRow,
+} from '@island.is/island-ui/core'
+import {
+  DataCategory,
+  TypeCategory,
+  AccessCategory,
+} from '@island.is/web/graphql/schema'
+import { capitalize } from './ServiceInformation'
+
+interface TagListProps {
+  data: Array<DataCategory>
+  type: Array<TypeCategory>
+  access: Array<AccessCategory>
+  namespace?: object
+}
+
+export const TagList: FC<TagListProps> = ({
+  data,
+  type,
+  access,
+  namespace,
+}) => {
+  const n = useNamespace(namespace)
+  return (
+    <Box paddingX={3}>
+      <GridContainer>
+        <GridRow>
+          <GridColumn
+            paddingBottom={[3, 3, 0]}
+            span={['12/12', '12/12', '6/12']}
+          >
+            <Box marginBottom={1}>
+              <Text variant="eyebrow" as="span" color="blue600">
+                {n('data')}
+              </Text>
+            </Box>
+            <Inline space={1}>
+              {data?.map((item) => (
+                <Tag variant="white" outlined key={item}>
+                  {n(`data${capitalize(item)}`)}
+                </Tag>
+              ))}
+            </Inline>
+          </GridColumn>
+          <GridColumn
+            paddingBottom={[3, 3, 0]}
+            span={['12/12', '12/12', '3/12']}
+          >
+            <Box marginBottom={1}>
+              <Text variant="eyebrow" as="span" color="blue600">
+                {n('type')}
+              </Text>
+            </Box>
```

Comment:
If its not required for this text to be `span` we could simplify this and use `Text` margins
```suggestion
            <Text variant="eyebrow"  color="blue600" marginBottom={1}>
              {n('type')}
            </Text>
```
`Text` is default `<p>` and that should have a default `display` value as `block`, unless I'm missing something here 😅

Reply:
This should apply to the same patterns where there is a single span inside Box